### ISSUE TITLE
Example is no longer working

### ISSUE THREAD

**Issue by llrs** [mvkorpel/bugtractr#1]
Hi, 
I had the same idea to analyze R bugs with R, and found your package! 
While playing with it I found an error with:

```r
library("bugtractr")
bi <- bug_info(1, "https://bugs.r-project.org/bugzilla/rest/bug/")
## Error in vapply(x, `[[`, "", "email") : values must be length 1,
##  but FUN(X[[1]]) result is length 0
```
This might be due to an API change. I've seen there hasn't been any commit in some time, but I leave this in case it is useful.

I had started developing a package that might use the API key to retrieve more information [here](https://github.com/llrs/bugRzilla), jut to learn how to use and authenticate in an API, and provide  more information to R users about R.

**Comment by mvkorpel** [comment]
Thank you! This project has been all but abandoned, but I think I should fix this. Good luck with your more ambitious project!

**Comment by mvkorpel** [comment]
This should be fixed by commit 5180875.
